hack/build-coredns.sh

```diff
@@ -4,16 +4,22 @@ set -euo pipefail
 
 PLUGIN_PATH=$(readlink -f "$(dirname "$0")/..")
 
-# Create a temporary directory for cloning openshift/coredns repo.
+# Create a temporary directory for cloning coredns repo.
```

Comment:
Can you please add a comment at the beginning of the script about the arg you can pass and its expected values?

Reply:
Added the comment explaining the argument being passed.